test/fixtures/cookbooks/kubernetes-stack-test/recipes/kubectl_install_default.rb

```diff
@@ -1 +1,3 @@
-kubectl 'install default kubectl'
+kubectl 'install default kubectl' do
```

Comment:
on "install default kubectl", we don't need to verify the installed version, just make sure that kubectl is installed is enough.